dotcom-rendering/src/components/Card/components/CardWrapper.tsx

```diff
@@ -58,7 +58,7 @@ const hoverStyles = css`
 	*/
 	:has(
 			ul.sublinks:hover,
-			.video-container.loop:hover,
+			.video-container:not(.cinemagraph):hover,
```

Comment:
Let's be more specific. Once long-form videos arrive we won't need to update this value